packages/amino/src/stdtx.spec.ts

```diff
@@ -50,3 +50,29 @@ describe("makeStdTx", () => {
     });
   });
 });
+
+describe("isStdTx", () => {
+  const validTx: StdTx = {
+    memo: "memoe",
+    msg: [{ type: "test", value: "Test Value" }],
+    fee: { amount: [{ denom: "ATOM", amount: "10" }], gas: "100000" },
+    signatures: [{ signature: "signature", pub_key: { type: "type", value: "value" } }],
+  };
+
+  it("should return true for a valid StdTx", () => {
+    expect(isStdTx(validTx)).toBeTruthy();
+  });
+
+  it("should return false for an invalid StdTx with missing properties", () => {
+    const { msg, ...invalidTx } = validTx;
```

Comment:
This is needed to make the linter happy

```suggestion
    const { msg: _msg, ...invalidTx } = validTx;
```